torch/fx/experimental/symbolic_shapes.py

```diff
@@ -4363,7 +4399,9 @@ def produce_guards_verbose(
         _simplified: bool = False,
         # Indicates if we should produce guards for known static values.
         ignore_static: bool = True,
-    ) -> Tuple[List[str], List[str]]:  # regular, verbose
+    ) -> Tuple[
+        List[str], List[str], Tuple[List[str], Dict[Source, sympy.Symbol]]
+    ]:  # python, verbose, cpp
```

Comment:
Aight, third time's a charm: you get to make a dataclass to store these results in :P